src/extras/primitives/primitives/a-video.js

```diff
@@ -0,0 +1,23 @@
+var materialMixin = require('../materialMixin');
+var registerPrimitive = require('../registerPrimitive');
+var utils = require('../../../utils/');
+
+registerPrimitive('a-video', utils.extendDeep({}, materialMixin(), {
+  defaultAttributes: {
+    geometry: {
+      primitive: 'plane'
+    },
+    material: {
+      color: '#FFF',
+      shader: 'flat',
+      side: 'double',
+      transparent: true
+    }
+  },
+
+  mappings: {
```

Comment:
I assume you want to force people to create `<video>` elements in `<a-assets>`. but keep in mind that removing `src` is a breaking change. and not to mention that makes most of these primitives pretty useless.